src/main/kotlin/inc/evil/d17map/services/ClassroomService.kt

```diff
@@ -58,6 +63,12 @@ class ClassroomService(
         return toClassroomResponse(updatedClassroom)
     }
 
+    fun getClassroomPhotoById(id: UUID): ByteArray? {
+        val classroom = classroomRepository.findById(id)
```

Comment:
Wydaje mi się, że mamy metodę findOne, która zamiast zwracać Optional zwraca Nullable obiekt. Nie jestem pewny czy to zostało zaimplementowane dla każdego entity, ale jest taki plik "template.kt", który ma tę metodę findOne